sentinelsat/sentinel.py

```diff
@@ -91,15 +91,18 @@ def _api2dhus_url(api_url):
     def _req_dhus_stub(self):
         try:
             resp = self.session.get(
-                self.api_url + "api/stub/version", auth=self.session.auth, timeout=self.timeout
+                self._api2dhus_url(self.api_url) + "api/stub/version",
+                auth=self.session.auth,
+                timeout=self.timeout,
             )
             resp.raise_for_status()
         except requests.exceptions.HTTPError as err:
+            # is this part of the code useful?
```

Comment:
Please remove - a question is better suited as a comment on your PR instead of in the code.

Reply:
Removing.